webapp/src/index.css

```diff
@@ -85,31 +85,39 @@ body {
   font-weight: 400;
 }
 
-/* colors */
-.color-warning {
+/* text colors */
+.text-color-warning {
   color: var(--color-warning);
 }
 
-.color-submit {
+.text-color-submit {
   color: var(--color-submit);
 }
 
-.color-online {
+.text-color-online {
   color: var(--color-online);
 }
 
-.color-offline {
+.text-color-offline {
   color: var(--color-offline);
 }
 
-.color-light {
+.text-color-light {
   color: var(--color-light);
 }
 
-.color-background {
+.text-color-background {
   color: var(--color-background);
 }
 
-.color-dark {
+.text-color-dark {
   color: var(--color-dark);
 }
+
```

Comment:
Why don't we like this in `webapp/src/shared/components/Text/Text.css` anymore?

Reply:
We move it here to reduce code duplication between the `Icon`, `Text`, and `Status` components because they share some parts.